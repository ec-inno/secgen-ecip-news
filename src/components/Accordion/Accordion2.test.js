import React from 'react';
import renderer from 'react-test-renderer';
import Accordion2 from './Accordion2';

describe('Accordion2', () => {
  const props = {
    children: 'test',
  };

  it('renders correctly', () => {
    const tree = renderer.create(<Accordion2 {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
