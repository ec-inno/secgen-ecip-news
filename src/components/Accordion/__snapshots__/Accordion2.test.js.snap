// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion2 renders correctly 1`] = `
<div
  className="ecl-accordion2"
  data-ecl-accordion2={true}
>
  test
</div>
`;
